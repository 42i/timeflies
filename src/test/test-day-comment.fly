day 2012-07-18 8 18
; aaa.bbb
day 2012-07-19 8 17
leave-days 2012-07-20 2012-07-30; Cycling in Greenland
day 2012-07-31, sick 8; snotty nose
day 2012-08-01 8 10, sick 4; snotty nose
off 2; parents' meeting
leave 2; star gazing
day 2012-08-02, phol; Troll's Day