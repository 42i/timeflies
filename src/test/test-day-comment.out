Time at work overview (all):
     when        worked   leave    sick    must
2012-07-18 Wed:   10.00 ----.-- ----.--    8.00
              ; aaa.bbb
2012-07-19 Thu:    9.00 ----.-- ----.--    8.00
2012-07-20 Fri: ----.--    8.00 ----.--    8.00 Cycling in Greenland
  week 2012-29:   19.00    8.00 ----.--   24.00
2012-07-23 Mon: ----.--    8.00 ----.--    8.00 Cycling in Greenland
2012-07-24 Tue: ----.--    8.00 ----.--    8.00 Cycling in Greenland
2012-07-25 Wed: ----.--    8.00 ----.--    8.00 Cycling in Greenland
2012-07-26 Thu: ----.--    8.00 ----.--    8.00 Cycling in Greenland
2012-07-27 Fri: ----.--    8.00 ----.--    8.00 Cycling in Greenland
  week 2012-30: ----.--   40.00 ----.--   40.00
2012-07-30 Mon: ----.--    8.00 ----.--    8.00 Cycling in Greenland
2012-07-31 Tue: ----.-- ----.--    8.00    8.00 snotty nose
 month 2012-07:   19.00   56.00    8.00   80.00
2012-08-01 Wed: ----.--    2.00    4.00    8.00 star gazing; snotty nose; parents' meeting
2012-08-02 Thu: ----.-- ----.-- ----.-- ----.-- Troll's Day
  week 2012-31: ----.--   10.00   12.00   24.00
 month 2012-08: ----.--    2.00    4.00    8.00
         total:   19.00   58.00   12.00   88.00
     when        worked   leave    sick    must
